Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -742,6 +742,15 @@ def audit_revision_and_version_scheme
     return if @new_formula
 
     fv = FormulaVersions.new(formula)
+
+    previous_version_and_checksum = fv.previous_version_and_checksum("origin/master")
+    [:stable, :devel].each do |spec_sym|
+      next unless spec = formula.send(spec_sym)
+      next unless previous_version_and_checksum[spec_sym][:version] == spec.version
+      next if previous_version_and_checksum[spec_sym][:checksum] == spec.checksum
+      problem "#{spec_sym}: only sha256 changed; needs to be confirmed by the developer"
```

Comment:
I still say we should link to the relevant documentation. Whatever the message, there will always be people that do not understand the wording. I argue it’s best to keep the message short and explain the why the error, not the reasoning behind it. That should be left to the longer documentation, where people are linked to. That way we’re not always hammering people with the long explanation and don’t need to keep updating the message to better clarify it (because it can be done at length in the documentation).